docs/tutorials/pi/make-a-plant-watering-robot.md

```diff
@@ -0,0 +1,483 @@
+---
+title: "How to Make a Plant Watering Robot with a Raspberry Pi"
+linkTitle: "Make a Plant Watering Robot with a Pi"
```

Comment:
```suggestion
title: "Plant Watering Robot with a Raspberry Pi"
linkTitle: "Watering Robot"
```

i want us to get away from having "How to" in almost all titles for tutorials. It doesn't add any useful info.